poreana/sample.py

```diff
@@ -220,11 +267,13 @@ def init_density(self, link_out, bin_num=150, remove_pore_from_res=True):
         remove_pore_from_res : bool, optional
             True to remove an extended pore volume from the reservoirs to only
             consider the reservoir space intersecting the crystal grid
+        bin_const_a : bool, optinal
+            If True all bins has the same surface. Otherwise the bins has the same bin width
```

Comment:
Englisch....

If true, all radial bins will have the same surface area, otherwise the bin-width will be constant

sollten wir eine optionale variable bin_width haben, für die Möglichkeit eine konstante binbreite festzulegen?